bakery/README.md

```diff
@@ -103,6 +103,15 @@ and
 
 will provide information about the command, defaults, and its positional and nonpositional arguments.
 
+When generating pipelines for the local environment, the following environment variables are used for credentials:
+
+| Environment variable | Required | Notes |
+| -------------------- | -------- | ----- |
+| `AWS_ACCESS_KEY_ID` | Yes | AWS credentials (likely for our sandbox) |
+| `AWS_SECRET_ACCESS_KEY` | Yes | AWS credentials (likely for our sandbox) |
```

Comment:
I put in empty aws credentials all the time if I'm not working on something that affects upload. I would say also not required for this one.

Reply:
Oh, haha, I do too. I guess I meant required to be set in your environment. I'll modify to say something to that effect, though.

Reply:
@tomjw64 - I changed the column header to try and make it clear. If it muddies the water too much and causes confusion, though, I can just remove the `Required` altogether.